xml/admin_operating_pools.xml

```diff
@@ -890,23 +890,47 @@ pool4                     1.1 MiB      13      0     39                  0
    need to migrate the pool data to another one whose parameters suit your
    deployment.
   </para>
-
   <para>
-   There are several methods of pool migration. We recommend using
-   <emphasis>cache tier</emphasis> because this method is transparent, reduces
-   cluster downtime, and avoids duplicating the entire pool's data.
+   There are two migration methods&mdash;a <emphasis>cache tier</emphasis>
+   method for general pool data migration, and a method using <command>rbd
+    migrate</command> sub-commands to migrate RBD images to a new pool. Each
+   method has its specifics and limitations.
   </para>
 
+  <sect2 xml:id="pool-migrate-limits">
+   <title>Limitations</title>
+   <itemizedlist>
+    <listitem>
+     <para>
+      You can use the cache tier method to migrate from a replicated pool to
+      either an EC pool or another replicated pool. Migrating from an
+      EC pool is not supported.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      You cannot migrate RBD images and &cephfs; exports from a replicated pool
+      to an EC pool. EC pools can store data but not metadata. The header
+      object of the RBD will fail to be flushed. The same applies for &cephfs;.
```

Comment:
Actually it is more than just the header object that will fail to migrate. There are other rbd objects that contain metadata (omap data) that will also fail to migrate. So I would change it to: "For example, the header object of the RBD will faill ...", or just skip this part.

Also, it probably worth mentioning that by metadata we mean any data stored as omap key-values, and EC does not support omap. So, instead of "EC pools can store data but not metadata." I would prefer something like: "Erasure coded pools do not support omap,  and RBD and cephfs use omap to store its metadata. But you can migrate data to EC pool, leaving metadata in replicated pool".